modules/core/src/main/java/org/apache/ignite/internal/managers/communication/TransmissionCancelException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.managers.communication;
+
+/**
+ * Exception is used to cancel a file transmission operation on the receiver. This exception may be thrown
+ * at anytime during session handling by a {@link TransmissionHandler} to gracefully interrupt the transmission
+ * session on a node-sender.
+ */
+public class TransmissionCancelException extends RuntimeException {
```

Comment:
Also, In my opinion, it's better to inherit this exception from the IgniteException.

Reply:
Fixed.